### DISCUSSION TITLE
Provide a switch to disable auto load of vaults

### DISCUSSION THREAD

**Discussion by masc23** [iterate-ch/cyberduck#15245] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
I want to use the duck-cli to synchronize a single folder with OneDrive. That folder happens to be a vault created with Cryptomator.

Everytime I try to use
`duck -u user@example.com --synchronize "onedrive:/My Files/Vault/" "C:\SomeDir\Vault"`,
duck-cli detects that the directory in question is a vault and tries some stuff, asking me for the password:
```
Unlock Vault. Provide your passphrase to unlock the Cryptomator Vault Vault.
Passphrase:
```
For me, it is not obvious why duck-cli needs to know the password for just synchronizing those two directories.

**Describe the solution you'd like**
I'd like to use a switch to tell duck-cli to just sync the files and don't do anything special with vaults.

E.g.:
`duck -u user@example.com --no-vault-autodetect --synchronize "onedrive:/My Files/Vault/" "C:\SomeDir\Vault"`,

**Describe alternatives you've considered**
Using Cyberduck GUI, which can be configured to ignore Cryptomator stuff. Not the best solution, unfortunately. CLI is better for automation purposes.

**Additional context**
For workflow reasons I use a local installation of Cryptomator to access the vault.

**Comment by dkocher** [comment, marked as answer]
The documentation shows how to [override default preferences](https://docs.cyberduck.io/cli/#preferences). To disable _Cryptomator_ support you can use

```
env "cryptomator.vault.autodetect=false" duck
```

**Comment by masc23** [comment]
Thank you, that works exactly as I wanted it to work :-)

Is there a list of these preferences? The docs mention a wiki, but I don't know where to find it.
And would there be an option to automatically choose "mirror" when synchronizing?

**Comment by dkocher** [comment]
> And would there be an option to automatically choose "mirror" when synchronizing?

Using `--existing mirror` should do the trick.

**Comment by dkocher** [comment]
> Is there a list of these preferences? 

Can be found in `default.properties` [^1].

[^1]: https://github.com/iterate-ch/cyberduck/blob/79bb79d9ac7f35ef809b97964f0c7c06dcead610/defaults/src/main/resources/default.properties#L4

**Comment by masc23** [comment]
Thank you very much for your help. I'm astonished on what short notice you helped.
Keep up the good work! 👍 


PS: From my side, the thread can be closed.
